From: Colin Ian King <colin.king@canonical.com>
Date: Mon, 10 Mar 2008 18:18:47 +0000 (+0000)
Subject: UBUNTU: QUIRK: fix Udma not fully available in Acer 1694 Wlmi
X-Git-Url: http://kernel.ubuntu.com/git?p=ubuntu%2Fubuntu-intrepid.git;a=commitdiff_plain;h=d28c48c14a5f898b64d290aaef711a620cdff257

UBUNTU: QUIRK: fix Udma not fully available in Acer 1694 Wlmi
OriginalAuthor: Tiago Sousa
OriginalLocation: http://launchpadlibrarian.net/11627664/new.ich_laptop.short.cables.diff
Bug: #187121

Add ICH6 on ACER Aspire 1694WLMi to list of laptops that use short cables rather than 80 wire

This should go upstream.
Signed-off-by: Colin Ian King <colin.king@canonical.com>
---

Index: 2008-devel/drivers/ata/ata_piix.c
===================================================================
--- 2008-devel.orig/drivers/ata/ata_piix.c
+++ 2008-devel/drivers/ata/ata_piix.c
@@ -723,6 +723,8 @@ static const struct ich_laptop ich_lapto
 	{ 0x27DF, 0x1043, 0x1267 },	/* ICH7 on Asus W5F */
 	{ 0x27DF, 0x103C, 0x30A1 },	/* ICH7 on HP Compaq nc2400 */
 	{ 0x24CA, 0x1025, 0x0061 },	/* ICH4 on ACER Aspire 2023WLMi */
+	{ 0x24CA, 0x1025, 0x003d },	/* ICH4 on Acer TM290 */
+	{ 0x266F, 0x1025, 0x0066 },	/* ICH6 on ACER Aspire 1694WLMi */
 	/* end marker */
 	{ 0, }
 };
